src/store/reducers/accounts.ts

```diff
@@ -808,6 +816,18 @@ export default ( state: AccountsState = initialState, action ): AccountsState =>
         return {
           ...state,
         }
+      case FETCH_RECEIVE_ADDRESS_SUCCEEDED:
+        return {
+          ...state,
+          receiveAddress: action.payload.data.rampReceiveAddress,
```

Comment:
This part might cause an issue, not sure tho. As the corresponding action doesn't specify a rampReceiveAddress property on it.

Reply:
Yes, that’s a mistake, it should be:
receiveAddress: action.payload.data.receiveAddress,